editor/project_manager/project_list.cpp

```diff
@@ -268,14 +286,53 @@ void ProjectListItemControl::set_unsupported_features(PackedStringArray p_featur
 					project_version_major = project_version_split[0].to_int();
 					project_version_minor = project_version_split[1].to_int();
 				}
-				if (GODOT_VERSION_MAJOR != project_version_major || GODOT_VERSION_MINOR <= project_version_minor) {
-					// Don't show a warning if the project was last edited in a previous minor version.
-					tooltip_text += TTR("This project was last edited in a different Godot version: ") + p_features[i] + "\n";
+
+				version_match_type = VER_PROJECT_IS_THIS_VER;
+				if (project_version_major > GODOT_VERSION_MAJOR) {
+					version_match_type = VER_PROJECT_USES_NEWER_MAJOR;
+				} else if (project_version_major < GODOT_VERSION_MAJOR) {
+					version_match_type = VER_PROJECT_USES_OLDER_MINOR;
+				} else {
+					// Project is same major version.
+					// Is it the same minor version, or an upgrade or downgrade?
+					if (project_version_minor > GODOT_VERSION_MINOR) {
+						version_match_type = VER_PROJECT_USES_NEWER_MINOR;
+					} else if (project_version_minor < GODOT_VERSION_MINOR) {
+						version_match_type = VER_PROJECT_USES_OLDER_MINOR;
+					}
 				}
+
+				if (version_match_type != VER_PROJECT_IS_THIS_VER) {
+					String project_version_tooltip_text = TTR("This project was last edited in a different Godot version: ") + p_features[i] + "\n";
+					if (version_match_type == VER_PROJECT_USES_OLDER_MAJOR || version_match_type == VER_PROJECT_USES_OLDER_MINOR) {
+						project_version_tooltip_text += vformat(TTR("Opening it will upgrade it to Godot %s.%s."), GODOT_VERSION_MAJOR, GODOT_VERSION_MINOR) + "\n";
+					} else if (version_match_type == VER_PROJECT_USES_NEWER_MAJOR || version_match_type == VER_PROJECT_USES_NEWER_MINOR) {
+						project_version_tooltip_text += vformat(TTR("Opening it will downgrade it to Godot %s.%s.\n"), GODOT_VERSION_MAJOR, GODOT_VERSION_MINOR);
+						project_version_tooltip_text += TTR("Downgrading projects is not recommended.\n");
```

Comment:
```suggestion
						project_version_tooltip_text += vformat(TTR("Opening it will downgrade it to Godot %s.%s."), GODOT_VERSION_MAJOR, GODOT_VERSION_MINOR) + "\n";
						project_version_tooltip_text += TTR("Downgrading projects is not recommended.\n");
```